contracts/LlamaLendFactory.sol

```diff
@@ -29,8 +29,13 @@ contract LlamaLendFactory is Ownable {
     }
 
     function emergencyShutdown(address[] calldata pools) external onlyOwner {
-        for(uint i = 0; i < pools.length; i++){
-            LendingPool(pools[i]).emergencyShutdown();
+        uint length = pools.length;
```

Comment:
i expect to never call this method, and if i call it will be one-off so gas doesnt matter much